source/mobile/mobile-faq.rst

```diff
@@ -4,26 +4,26 @@ Mobile Apps FAQ
+At the moment, we only support connecting to one server at a time; however, we are aware that this is one of the `top feature requests <https://mattermost.uservoice.com/forums/306457-general/suggestions/10975938-ios-and-android-apps-should-allow-multiple-server>`__ for the mobile app. We are currently investigating some technical challenges, such as how to handle push notifications coming from multiple servers. To follow our progress on this feature, you can join the `RN: Multi-Server <https://community.mattermost.com/core/channels/rn-multi-server-suppot>`_ channel on our community server.
 
-As a workaround, you can install both the released "Mattermost" app and sign up to be a `tester <https://github.com/mattermost/mattermost-mobile/blob/master/README.md#testing>`__ for the "Mattermost Beta" app. This allows you to connect and log in to a different server from each app.
+As a workaround, you can install the released Mattermost mobile app and sign up to be a `tester <https://github.com/mattermost/mattermost-mobile/blob/master/README.md#testing>`__ for the Mattermost Beta mobile app. This allows you to connect and log in to a different server from each app.
 
 Is there a tablet version of the mobile apps?
 ---------------------------------------------
 
-“Mattermost Classic” mobile apps support tablets.
+Mattermost Classic mobile apps support tablets.
 
-Our second generation mobile apps (“Mattermost”) have beta support for tablets.
+Our second generation mobile apps (Mattermost) have beta support for tablets.
 
 Can the permanent sidebar on tablet devices be disabled?
 --------------------------------------------------------
 
-The permanent sidebar is on by default for tablet sized devices, but it can be disabled from **Settings > Display > Sidebar > Permanent Sidebar**. When disabled, the sidebar behaves similarly to mobile devices where the user must open it using the button in the top-left of the screen.
+The permanent sidebar is on by default for tablet-sized devices, but can be disabled from **Settings > Display > Sidebar > Permanent Sidebar**. When disabled, the sidebar behaves similarly to mobile devices where the user must open it using the button in the top-left corner of the screen.
 
 How is data handled on mobile devices after a user account is deactivated?
 --------------------------------------------------------------------------
 
-App data is wiped from the device when a user logs out of the app. If the user is logged in when the account is deactivated, then within one minute the system logs the user out, and as a result all app data is wiped from the device.
+App data is wiped from the device when a user logs out of the app. If the user is logged in when the account is deactivated, then within one minute of deactivation the system logs the user out. Thereafter all app data is wiped from the device.
```

Comment:
@DSchalla / @esethna, confirming that this is correct.

Reply:
Yes, the only caveat is that if the user is logged in but not using the app, they'll get logged out the next time the open the app and therefore all data is wiped.